pyquil/api/_compiler.py

```diff
@@ -375,13 +210,17 @@ def native_quil_to_executable(
         if response.ro_sources == []:
             response.ro_sources = _collect_classical_memory_write_locations(nq_program)
 
-        if not debug:
```

Comment:
Should we keep this? There's no `debug` field on the response, but I'm guessing we're just patching it in like we do for the `recalculation_table` above.